sql/item_mods.sql

```diff
@@ -52181,6 +52181,12 @@ INSERT INTO `item_mods` VALUES (23795,1,1); -- DEF: 1
 -- Ziamet Nails
 INSERT INTO `item_mods` VALUES (23796,1,1); -- DEF: 1
 
+-- Poroggo Cass. +1
```

Comment:
These resistances and damage null only happens on watersday. Putting them here makes them always on. Latents can have qualifiers, so maybe it is better to have these there. 
> Watersday: Haste+25% All ailment resistance +20 Occasionally ignores damage taken

You already have the haste properly addressed, but these are not item_mods, because they are not always active.